api/jsonrpc/server.go

```diff
@@ -204,7 +204,7 @@ func (j *JSONRPCServer) ExecuteActions(
 
 	for actionIndex, action := range actions {
 		// Get expected state keys
-		stateKeysWithPermissions := action.StateKeys(args.Actor)
+		stateKeysWithPermissions := action.StateKeys(args.Actor, ids.Empty)
```

Comment:
Can we use `CreateActionID(ids.Empty, actionIndex)` here? This ensures that each simulated action gets a unique `actionID` during the simulation and that it will never overlap with an actual `actionID` that was already used onchain.

Reply:
Done